README.md

```diff
@@ -8,7 +8,7 @@ The art and idea of coding is an impactful form of demonstration in todays time.
 
 Remember that as you undertake this journey you will begin to share,edit,create and update the ideas that exsist in your inner thoughts and hearts.
```

Comment:
ok here is a sample comment..but how do I add an emoji :neckbeard:  lol ohh I see :heartpulse: